jdaviz/app.py

```diff
@@ -2032,12 +2032,19 @@ def vue_data_item_remove(self, event):
         orientation_plugin = self._jdaviz_helper.plugins.get("Orientation")
         if orientation_plugin is not None:
             orient = orientation_plugin.orientation.selected
+            if orient == data_label:
+                orient = "Default orientation"
             self._reparent_subsets(data, new_parent=orient)
         else:
             self._reparent_subsets(data)
 
-        # Make sure the data isn't loaded in any viewers
+        # Make sure the data isn't loaded in any viewers and isn't the selected orientation
         for viewer_id in self._viewer_store:
+            if orientation_plugin is not None:
+                orientation_plugin.viewer.selected = viewer_id
+                orient = orientation_plugin.orientation.selected
+                if orient == data_label:
+                    orientation_plugin.orientation.selected = "Default orientation"
```

Comment:
Instead of hardcoding this string, use this var: https://github.com/spacetelescope/jdaviz/blob/deda388472169d11604db304a369869e2e13d00e/jdaviz/configs/imviz/helper.py#L22

Reply:
Done!